ProjFS.Mac/PrjFS.xcodeproj/project.pbxproj

```diff
@@ -119,13 +119,13 @@
 		};
 		4A08256621E77B7F00E21AFD /* CopyFiles */ = {
 			isa = PBXCopyFilesBuildPhase;
-			buildActionMask = 2147483647;
-			dstPath = /Library/LaunchDaemons;
-			dstSubfolderSpec = 0;
+			buildActionMask = 12;
+			dstPath = "";
+			dstSubfolderSpec = 16;
```

Comment:
What do `buildActionMask` and `dstSubfolderSpec` control?

Reply:
These are values that were computed by me making changes in the GUI.

Before it was configured to copy the org.vfsforgit.prjfs.PrjFSKextLogDaemon.plist into /usr/local/libexec when running 'Install' from the IDE. Since we don't install our app that way, I reused this build rule to have the .plist copied into the BuildOutput folder next to the kext and daemon. Before it wasn't being copied to BuildOutput at all and we need it in there for the installer.

Reply:
Note that the "install location" setting in Xcode will not actually install to that absolute location but rather copy into that subpath of the final build artefact directory when running an `xcodebuild install`.

In other words, if you set all the install paths up correctly, you can get `xcodebuild install` to prepare everything in a directory structure that's ready for packaging up.

Reply:
Oh neat! I didn't know `xcodebuild install` was a thing, I learned something new today! I think our current approach works fine so I'm not too inclined to rework how the directory structure is created, but this is good to know for the future.